packages/node_modules/nav-frontend-skjema/src/input.tsx

```diff
@@ -37,7 +37,7 @@ export interface NavFrontendInputProps extends React.InputHTMLAttributes<HTMLInp
     /**
      * Referanse til selve inputfeltet. Brukes for eksempel til å sette fokus
      */
-    inputRef?: () => any;
+    inputRef?: (element: HTMLInputElement | null) => any;
```

Comment:
Hm, lurer på om typen her ikke heller burde være:
```tsx
inputRef?: React.RefObject<HTMLInputElement>;
```
Som da brukes slik:

```tsx
private referanse = React.createRef<HTMLInputElement>();

<Input inputRef={referanse} />
```

Reply:
Jeg vet ikke. Men `inputRef?: (element: HTMLInputElement | null) => any;` fungerer.

Reply:
Jau, men `RefObject` er allerede definert som `<T> | null`. Tenker det er ryddigere å bruke mer spesifikke typer hvis de finnes - i stedet for slike selv-komponerte `() => any`.

Reply:
`React.createRef` finnes ikke i react før 16.3. Så da må evt alle peerDependencies også oppdateres.

Reply:
Aha 👍